di-ipv-core-stub/src/main/java/uk/gov/di/ipv/stub/core/CoreStub.java

```diff
@@ -40,6 +40,7 @@ private void initRoutes() throws Exception {
         Spark.get("/evidence-request", coreStubHandler.evidenceRequest);
         Spark.get("/authorize", coreStubHandler.authorize);
         Spark.get("/user-search", coreStubHandler.userSearch);
+        Spark.post("/user-search", coreStubHandler.sendSharedClaim);
```

Comment:
Just to be a bit cleare can we change the name of this method to sendRawSharedClaim?